src/backend/identities.rs

```diff
@@ -345,33 +355,42 @@ impl AppState {
                 let loaded_identity = self.loaded_identity.lock().await;
                 if let Some(identity) = loaded_identity.as_ref() {
                     transfer_transition.set_identity_id(identity.id());
-                    let nonce = sdk
-                        .get_identity_nonce(identity.id(), true, None)
-                        .await
-                        .expect("Expected to get an identity nonce in creating credit transfer");
+                    let nonce = match sdk.get_identity_nonce(identity.id(), true, None).await {
```

Comment:
maybe `let Ok(nonce) = ... else { return ...};` ?

Reply:
But then you can't include the error message from sdk

Reply:
ah really, you're right, didn't notice you use it

disregard it please, and the rest of similar comments